core/src/main/scala/org/apache/spark/scheduler/DAGSchedulerEvent.scala

```diff
@@ -65,10 +65,13 @@ private[scheduler] case class JobCancelled(
 
 private[scheduler] case class JobGroupCancelled(
     groupId: String,
+    reason: Option[String],
```

Comment:
ditto

Reply:
Thank you for the suggestions, just addressed all the comments above.